GenderPayGap.WebUI/Models/Admin/AdminChangeCompaniesHouseOptInOutViewModel.cs

```diff
@@ -15,7 +15,7 @@ public class AdminChangeCompaniesHouseOptInOutViewModel
         public CompaniesHouseCompany CompaniesHouseCompany { get; set; }
 
         [GovUkValidateRequired(ErrorMessageIfMissing = "Please enter a reason for this change.")]
-        [GovUkValidateCharacterCount(MaxCharacters = 250)]
+        [GovUkValidateCharacterCount(MaxCharacters = 250, NameAtStartOfSentence = "Reason", NameWithinSentence = "Reason")]

```

Comment:
This is the sort of thing I was asking about in the [review for GPG-806](https://github.com/cabinetoffice/gender-pay-gap/pull/467) 😄

Reply:
Indeed it was!